### PR TITLE
DBZ-5249 KafkaDatabaseHistory without check database history topic create result caused UnknowTopicOrPartitionException

### PR DESCRIPTION
**Pull request by smallYellowCat**
https://issues.redhat.com/browse/DBZ-5249

KafkaDatabaseHistory without check database history topic create result caused UnknowTopicOrPartitionException

### CODE REVIEW

File: debezium-storage/debezium-storage-kafka/src/main/java/io/debezium/storage/kafka/history/KafkaDatabaseHistory.java [hunk 1]
```diff
@@ -532,7 +533,8 @@ public void initializeStorage() {
 
             topic.configs(Collect.hashMapOf(CLEANUP_POLICY_NAME, CLEANUP_POLICY_VALUE, RETENTION_MS_NAME, Long.toString(RETENTION_MS_MAX), RETENTION_BYTES_NAME,
                     Long.toString(UNLIMITED_VALUE)));
-            admin.createTopics(Collections.singleton(topic));
+            CreateTopicsResult result = admin.createTopics(Collections.singleton(topic));
+            result.all().get(30L, TimeUnit.SECONDS);
```
Comment (Naros) [inline debezium-storage/debezium-storage-kafka/src/main/java/io/debezium/storage/kafka/history/KafkaDatabaseHistory.java:537]: Could we define a `Field` called `kafka.topic.create.timeout.ms` that behaves similarly to the existing field called `kafka.query.timeout.ms`?  I think the 30-second delay is acceptable, but I'd prefer this to be something a user can tweak on an as-needed basis. In addition, the `ref-connector-configuration-database-history-properties.adoc` documentation partial should be updated to include this new property.

**Review by Naros — COMMENTED** [review]
Hi @smallYellowCat, I've added a comment inline.

File: debezium-storage/debezium-storage-kafka/src/main/java/io/debezium/storage/kafka/history/KafkaDatabaseHistory.java (hunk 1 above)
Comment (smallYellowCat) [reply debezium-storage/debezium-storage-kafka/src/main/java/io/debezium/storage/kafka/history/KafkaDatabaseHistory.java]: ok, thank u for your sugestions, i will fix it.

**Review by Naros — APPROVED** [review]

**Comment by Naros** [comment]
Thanks, @smallYellowCat, applied to main (=2.0).

**Pull request merged**